cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -222,8 +222,35 @@ parameters:
 - name: CS_SERVICE_PRINCIPAL_CREDS_BASE64
   description: Base64 encoded service principal credentials for CS. This is temporary for P1
   required: true
+- name: KEYVAULT_NAME
+  description: The name of the Key Vault where the first party credentials are stored.
+  value: "service-kv-aro-hcp-dev"
+- name: TENANT_ID
+  description: The Tenant ID of Key Vault.
+  value: ""
+
 objects:
 
+- apiVersion: secrets-store.csi.x-k8s.io/v1
+  kind: SecretProviderClass
+  metadata:
+    name: cs-keyvault
```

Comment:
name may need to change based on how we're organising keyvault identities. It may be the case we'll need multiple SecretProviderClass if we have multiple keyvaults to integrate with cs.

Reply:
Related to the same questions above.

Reply:
We'll have multiple key vaults, but I believe this is the only one which will be used to mount a secret in the cluster for CS usage.  The others CS will access the data plane of the key vaults directly, rather than leveraging secrets CSI driver.

Reply:
i'm fine with leaving it with this name then, feel free to close thread as resolved 👍

Reply:
I can use something like `cs-azure-id-credentials-keyvault` to specify what the keyvault is for.